app/serializers/user_serializer.rb

```diff
@@ -31,7 +31,7 @@ def services
       BackendServices::EDUCATION_BENEFITS
     ]
     service_list += [BackendServices::RX, BackendServices::MESSAGING] if object.can_access_mhv?
-    service_list << BackendServices::DISABILITY_BENEFITS if object.can_access_evss?
+    service_list << BackendServices::EVSS_BENEFITS if object.can_access_evss?
```

Comment:
This will break the frontend immediately once its merged. Could we add the new service name as well as the old one to the list? Also, not sure what the best service name is here, maybe `claim-status`?